src/compiler/python_generator.cc

```diff
@@ -229,7 +229,8 @@ bool GetModuleAndMessagePath(const Descriptor* type,
   return true;
 }
 
-bool PrintServerFactory(const ServiceDescriptor* service, Printer* out) {
+bool PrintServerFactory(const std::string package_qualified_service_name,
```

Comment:
It means pass an immutable reference instead of the (current) pass an immutable copy - which is a good habit to get into. Performance isn't a huge concern in this code, but I've seen some really small codebases slow to a glacial speed with unnecessary copying in the past, so it's worth getting this detail right.